db/seeds.rb

```diff
@@ -11,11 +9,787 @@
+while start_date <= end_date
+  limit = view_count + rand(rand(20..30)..rand(60..100))
+  while view_count < limit
+    Viewcount.create(id: view_count, deal_id: 1003, user_id: view_count, created_at: start_date)
+    view_count = view_count + 1
+  end
+  start_date = start_date + 1
+end
+num_view_count = view_count - starting_view_count
+unique_view_count = num_view_count- rand(50..90)
+DealAnalytic.create(id: 1003, deal_id: 1003, view_count: num_view_count, unique_view_count: unique_view_count, redemption_count: num_redemption)
+
+# Seed DealAnalytics for View count for deal 1004
+starting_redemption = redemption
+start_date = DateTime.parse("2015-10-19 00:00:00")
+end_date = DateTime.parse("2015-12-01 00:00:00")
+while start_date <= end_date
+  limit = redemption + rand(0..rand(0..30))
+  venue_id = rand(1000..1002)
+  while redemption < limit
+    Redemption.create(id: redemption, deal_id: 1004, user_id: redemption, venue_id: venue_id, created_at: start_date)
+    redemption = redemption + 1
+  end
+  start_date = start_date + 1
+end
+num_redemption = redemption - starting_redemption
+
+starting_view_count = view_count
+start_date = DateTime.parse("2015-10-15 00:00:00")
+end_date = DateTime.parse("2015-12-01 00:00:00")
+while start_date <= end_date
+  limit = view_count + rand(rand(10..30)..rand(40..50))
+  while view_count < limit
+    Viewcount.create(id: view_count, deal_id: 1004, user_id: view_count, created_at: start_date)
+    view_count = view_count + 1
+  end
+  start_date = start_date + 1
+end
+num_view_count = view_count - starting_view_count
+unique_view_count = num_view_count- rand(50..90)
+DealAnalytic.create(id: 1004, deal_id: 1004, view_count: num_view_count, unique_view_count: unique_view_count, redemption_count: num_redemption)
+
+# Seed DealAnalytics for View count for deal 1011
+starting_redemption = redemption
+start_date = DateTime.parse("2015-03-01 00:00:00")
+end_date = DateTime.parse("2015-12-01 00:00:00")
+while start_date <= end_date
+  limit = redemption + rand(0..30)
+  venue_id = rand(1000..1002)
+  while redemption < limit
+    Redemption.create(id: redemption, deal_id: 1011, user_id: redemption, venue_id: venue_id, created_at: start_date)
+    redemption = redemption + 1
+  end
+  start_date = start_date + 1
+end
+num_redemption = redemption - starting_redemption
+
+starting_view_count = view_count
+start_date = DateTime.parse("2015-02-25 00:00:00")
+end_date = DateTime.parse("2015-12-01 00:00:00")
+while start_date <= end_date
+  limit = view_count + rand(20..40)
+  while view_count < limit
+    Viewcount.create(id: view_count, deal_id: 1011, user_id: view_count, created_at: start_date)
+    view_count = view_count + 1
+  end
+  start_date = start_date + 1
+end
+num_view_count = view_count - starting_view_count
+unique_view_count = num_view_count- rand(50..90)
+DealAnalytic.create(id: 1005, deal_id: 1011, view_count: num_view_count, unique_view_count: unique_view_count, redemption_count: num_redemption)
```

Comment:
Yup! 1005 and 1006 are waiting deals. The rest are not really required for the data